06-building-a-program-start-to-finish/writing-tests/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: Mocha/Jasmine with any assertion library of choice (Chai.JS, Should.JS, Expect.JS...)
+- **Deliver**: a Pull Request showing all tests required for your project
+- **With**: *independently*
+
+#### Link to Pull Request for Tests
+- [Click here to view the PR](https://github.com/andela-aawa/inverted-index/pull/17)
```

Comment:
@q-ode I have made all changes as requested. Please have a look at them. I need to merge my branches and move on.